src/vs/editor/browser/controller/input/textAreaWrapper.ts

```diff
@@ -29,9 +29,12 @@ class ClipboardEventWrapper implements IClipboardEvent {
 		return false;
 	}
 
-	public setTextData(text: string): void {
+	public setTextData(text: string, richText?: string): void {
```

Comment:
I would use null for richText as a signal that it should not be set (instead of undefined + optional).

IMHO null is a lot more clear and prevents bugs where we forget by accident to add a second argument....